### ISSUE TITLE
Investigate using accessibility info for missing performances

### ISSUE THREAD

**Issue by alistairjcbrown** [clusterflick/scripts#264]
Review what we currently do if there are performances on the accessibility data site that we don't have. I think we ignore them unless they have accessibility info - but flagging these could be interesting.

Are there a lot?
Do they have access to performances but public on the site?
